src/presentation/controller/global/Controller.ts

```diff
@@ -31,6 +32,41 @@ class GlobalController extends AbstractController {
     };
   }
 
+  getSignOut(): RequestHandler {
+    return (request: Request, response: Response, next: NextFunction) => {
+      try {
+        const previousPageUrl = this.getPreviousPageUrl(request);
+        const pageType = super.pageType(request.path);
+        const pageRouting = super.getRouting(
+          registrationsRouting,
```

Comment:
i think the sign-out page should live in the global routing rather than registrationsRouting as it will be applicable to all journeys (registration, transition and others)

Reply:
actually i can see it in the global routing file so should use global routing instead of registrationRouting in the call to super.getRouting

Reply:
I have had to change this for the start page as well, registrations routing should probably not be in this controller at all.